neuralmonkey/readers/image_reader.py

```diff
@@ -27,6 +33,13 @@ def image_reader(prefix="",
         pad_h x pad_w x number of channels.
     """
 
+    if (((rescale_w and not rescale_h) or (not rescale_w and rescale_h))
+            and not keep_aspect_ratio):
+        raise ValueError(
+            "While rescaling only one side, aspect ratio must be kept, "
+            "was set to false.")
+
+    # pylint: disable=too-many-branches
```

Comment:
nechápu, v tomhle bloku jsi ubral jednu větev, jakto že je tam najednou too many branches? A je-li to oprávněně, nejde to rozfunkcovat? tadyten pylint disable je podle mě nedostatečně opodstatněnej